### PR TITLE
minimum OP

### PR DESCRIPTION
**Pull request by megawidget**
bonus: cleaned up tests to use `import ngraph` and fixed the mac build :trollface:

### CODE REVIEW

File: python/CreatePipWhl.cmake [hunk 1]
```diff
@@ -53,6 +53,30 @@ if (PYTHON)
         # 1. Get the NGRAPH_VERSION exported from nGraph CMake
         # 2. If #1 isn't possible, then determine this by looking at the 
         # dependency list?
+        execute_process(COMMAND
```
Comment (aprocter) [inline python/CreatePipWhl.cmake]: Looks like this command is repeated below (just past the greentext).

File: test/python/conftest.py [hunk 2]
```diff
@@ -22,12 +21,7 @@
 
 from tensorflow.python.framework import ops
 
-from common import LIBNGRAPH_DEVICE
-
-
-@pytest.fixture(scope='session', autouse=True)
-def load_ngraph_device():
-  cdll.LoadLibrary(os.path.join('../../src', LIBNGRAPH_DEVICE))
+import ngraph  # noqa
```
Comment (avijit-nervana) [inline test/python/conftest.py]: We cannot do this at the unit test level. The unit tests must not require the Python package (which is built using `make install` that logically comes after running the tests and ensuring they all pass. Please revert his change.

Comment (megawidget) [reply test/python/conftest.py]: Ack.  Okay.

File: python/CreatePipWhl.cmake (hunk 1 above)
Comment (megawidget) [reply python/CreatePipWhl.cmake]: Yes, with different arguments. :|

Comment (aprocter) [reply python/CreatePipWhl.cmake]: Ooh, there it is. `libcpu_backend` vs `libngraph_device`. Okey dokey.

Comment (avijit-nervana) [inline python/CreatePipWhl.cmake]: While you are at it - would you mind parameterize the version numbers i.e., `.0.4.0` --> NGRAPH_REL_VERSION or something. For now we will hardcode this but in future - we will figure out a way to get this from nGraph programmatically. Thanks :)

Comment (avijit-nervana) [reply python/CreatePipWhl.cmake]: Actually nevermind. I am doing a PR to update the nGraph version - I will do this as part of that.

**Pull request merged**